presto-hive/src/main/java/com/facebook/presto/hive/metastore/thrift/ThriftHiveMetastoreConfig.java

```diff
@@ -0,0 +1,75 @@
+package com.facebook.presto.hive.metastore.thrift;
+
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+import io.airlift.units.Duration;
+
+import javax.validation.constraints.NotNull;
+
+public class ThriftHiveMetastoreConfig {
```

Comment:
Please add a config test class `TestThriftHiveMetastoreConfig`. See `TestHiveS3Config` for an example.

Reply:
@electrum  added;